README.md

```diff
@@ -23,8 +23,7 @@ $ git flow release publish 0.1.0
 $ git flow release finish 0.1.0
 ```
 
-To kick off the deployment, you'll still need to push the local tags remotely
-`git push --tags`
+After you've completed the `git flow` steps, `git push develop` and `git push master` to sync your local branches with the remote repo. The last step is to `git push --tags` to push local tags to remote and deploy the release via Travis-CI.
```

Comment:
Depending on the person's local Git configuration, the `git push <branch>` behavior may require additional steps. It might need to be something like:

```bash
$ git checkout develop
$ git push origin develop
$ git checkout master
$ git push origin master
```

Even that might be different because people may have tweaked their default `origin`.